packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -722,8 +722,13 @@
   "Chat_closed_successfully": "Chat closed successfully",
   "Chat_History": "Chat History",
   "Chat_Now": "Chat Now",
+  "chat_on_hold_due_to_inactivity": "This chat is on-hold due to inactivity",
+  "Chat_On_Hold": "Chat On-Hold",
+  "Chat_On_Hold_Successfully": "This chat was successfully placed On-Hold",
   "Chat_queued": "Chat Queued",
   "Chat_removed": "Chat Removed",
+  "Chat_resumed": "Chat Resumed",
+  "Chat_resumed_successfully": "Chat Resumed Successfully from On-Hold",
```

Comment:
I didn't find where this key is used.

Reply:
You're right, its not getting used anywhere. Removing it